.eslintrc.js

```diff
@@ -0,0 +1,90 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  parser: 'babel-eslint',
+  extends: [
+    'airbnb',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+    createjs: false,
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    'array-bracket-newline': 'warn',
+    'no-underscore-dangle': 'off',
+    'no-param-reassign': 'off',
+    'import/extensions': [{ js: 'always', json: 'never' }],
+    'no-plusplus': 'off',
+    'object-curly-newline': 'warn',
+    'class-methods-use-this': 'off',
+    'no-unused-expressions': 'off',
+    'no-use-before-define': 'error',
+    'object-shorthand': 'error',
+    'no-multi-spaces': 'warn',
+    'arrow-spacing': ['error', { before: true, after: true }],
+    'space-in-parens': 'warn',
+    'consistent-return': 'off',
+    'no-mixed-operators': 'warn',
+    'no-bitwise': 'warn',
+    'no-multiple-empty-lines': 'warn',
+    'no-new': 'warn',
+    'no-eval': 'warn',
+    'no-lone-blocks': 'off',
+    'no-return-assign': 'warn',
+    'no-duplicate-case': 'off',
```

Comment:
This seems like one that we would want on and I'm not seeing it mentioned in your previous PR, so I'm not sure how this one got here.

Reply:
updated